### DISCUSSION TITLE
Temperatures from Techem FHKV data II/III (fhkvdataiii) very off - normal?

### DISCUSSION THREAD

**Discussion by muhkuhgh** [wmbusmeters/wmbusmeters#691] [General]
Hello everyone,

I just bought a suitable USB Stick and after some complications I've got this set up and running tonight and my current HCAs are already being stored in mySQL database.

I find some readings very disturbing.. most of my rooms are like 20 °C and the water temperature for heating is below 70 °C in our house afaik.

Are your temperature values as worse and unusable as mine?
![image](https://user-images.githubusercontent.com/38386384/203418455-3f78fd7f-3820-4f69-bdbd-832e87c8dfdd.png)
7 different devices.
118 °C .. seriously? :D I mean that would explain the already 1600 hca for this radiator this year..

Also: I guess it is not possible to get decimals for HCA as they are probably just using internal but not send out - right?

**Comment by weetmuts** [comment]
If you capture a couple of telegrams with --logtelegrams then post them here, then we can check the contents of the telegrams. You can also use https://wmbusmeters.org/ to analyze them.
